mayastor/src/replica.rs

```diff
@@ -281,7 +281,7 @@ impl Replica {
     }
 
     /// Create a snapshot
-    pub fn create_snapshot(
+    pub async fn create_snapshot(
```

Comment:
any reason for adding async, seems the function itself is not doing anything async?

Reply:
The caller seemed to require it to be async.